common/pkcs11.h

```diff
@@ -12,12 +12,12 @@
    the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
    PURPOSE.  */
 
-/* Please submit changes back to the Scute project at
-   http://www.scute.org/ (or send them to marcus@g10code.com), so that
+/* Please submit any changes back to the p11-kit project at
+   https://github.com/p11-glue/p11-kit/, so that
```

Comment:
It's up to you. I think that scute has more limited scope of pkcs11 (gnupg cards) thus they may have no reason to support new mechanisms, though they could benefit from the pkcs11 file here also. The fork of that file goes way back to past and I'm not sure that this file is being updated since then:
https://github.com/gpg/scute/blob/master/src/pkcs11.h